_posts/0000-01-02-drhrad.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```

Comment:
I would change the graphic design of the menus to more easily see the progress the own work and the preformed activities and text

Reply:
Yes if only I would understand why this software should help me and for which part of my work.:-))

Reply:
okay the final comment without knowing